package/MDAnalysis/core/topologyattrs.py

```diff
@@ -32,21 +32,122 @@
 """
 
 from six.moves import zip, range
+
+import Bio.Seq
+import Bio.SeqRecord
+import Bio.Alphabet
```

Comment:
Biopython used to be imported inside of a class (https://github.com/MDAnalysis/mdanalysis/pull/1164/files#diff-96c7f631815d065c6f087f4aea01a4e6L1035). The PR is just moving the imports at the top.